trustymail/cli.py

```diff
@@ -22,6 +23,14 @@
   --dmarc                     Only check dmarc records
   --json                      Output is in json format (default csv)
   --debug                     Output should include error messages.
+  --dns-hostnames=HOSTNAMES   A comma-delimited list of DNS servers to query 
```

Comment:
Consider renaming `dns-hostnames` just `dns`. In most instances, IPs will be entered, not actual hostnames, and `dns` is faster to type on the command-line.